libraries/Microsoft.Bot.Schema/Teams/TeamsPagedMembersResult.cs

```diff
@@ -30,7 +30,7 @@ public TeamsPagedMembersResult(string continuationToken = default, IList<Channel
         {
             ContinuationToken = continuationToken;
             var teamsChannelAccounts = members.Select(channelAccount => JObject.FromObject(channelAccount).ToObject<TeamsChannelAccount>());
-            Members = teamsChannelAccounts.ToList();
+            Members = teamsChannelAccounts.ToList() ?? new List<TeamsChannelAccount>();
```

Comment:
If teamsChannelAccounts is null, this will fail and the code after the ?? will never be executed
```suggestion
            Members = teamsChannelAccounts?.ToList() ?? new List<TeamsChannelAccount>();
```

Reply:
Changed applied. Thanks!